### DISCUSSION TITLE
[Question] I am having stepper block for 0.5sec on printers turning on

### DISCUSSION THREAD

**Discussion by fpmirabile** [Jyers/Marlin#1051] [Q&A] [answered]
Hey guys,

So a quick story about what is happening:
I broke my stock motherboard (4.2.2) with a short circuit, so I bought 4.2.7.
I compiled jyers changing the motherboard variable and everything works smoothly.

The only thing I am noticing, and Idk if this comes from jyers or is the dumb motherboard: Every time I turn on the printers is like the motors starts on, like when you go to Prepare => Move => and move any axis.
It is less than a second, and then they work fine.
I notice this too, because sometimes if  you move the bed with printer off (I do this slowly but noticeable anyway) the steppers motors block the movement.

Any idea where should I take a look ?

**Comment by Cal-LF** [comment, marked as answer]
The steppers will thump when printer is turned on.
There will be stepper and belt drag when you move the bed with the printer turned off but should move smoothly.
This is not a Jyres firmware issue as it is not even loaded yet when turned on or when the printer is powered off.
